ckan/migration/versions/081_set_datastore_active.py

```diff
@@ -0,0 +1,48 @@
+import json
+from sqlalchemy import create_engine
+from sqlalchemy.sql import text
+from pylons import config
+
+
+def upgrade(migrate_engine):
+
+    datastore_connection_url = config.get(
+        'ckan.datastore.read_url', config.get('ckan.datastore.write_url'))
+
+    if not datastore_connection_url:
+        return
+
+    datastore_engine = create_engine(datastore_connection_url)
+
+    try:
+
+        resources_in_datastore = datastore_engine.execute('''
+            SELECT table_name
+            FROM information_schema.tables
+            WHERE table_schema = 'public'
+        ''')
+
+        resources = migrate_engine.execute('''
+            SELECT id, extras
+            FROM resource
+            WHERE id IN ({0}) AND extras IS NOT NULL
+        '''.format(
+            ','.join(['\'{0}\''.format(_id[0])
+                      for _id
+                      in resources_in_datastore])
+            )
+        )
```

Comment:
@wardi @joetsoi any ideas on how to use parameters binding for this (in an IN clause)?